samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -422,15 +422,18 @@ static void errata_117(nrf_radio_mode_t mode)
 		*((volatile uint32_t *)0x41008588) = *((volatile uint32_t *)0x01FF0080);
 	}
 }
-#endif /* NRF53_ERRATA_117_ENABLE_WORKAROUND */
+#else
+static void errata_117(nrf_radio_mode_t mode)
+{
+	ARG_UNUSED(mode);
+}
+#endif /* NRF53_ERRATA_117_PRESENT */
 
 static void radio_mode_set(NRF_RADIO_Type *reg, nrf_radio_mode_t mode)
 {
-#if NRF53_ERRATA_117_ENABLE_WORKAROUND
 	if (nrf53_errata_117()) {
```

Comment:
Why does this errata have different schema of handling it than others? In other erratas you check inside the handling function.

Reply:
Corrected